src/moveit_visual_tools.cpp

```diff
@@ -569,15 +575,15 @@ bool MoveItVisualTools::publishIKSolutions(const std::vector<trajectory_msgs::Jo
   for (std::size_t i = 0; i < ik_solutions.size(); ++i)
   {
     trajectory_pt_timed = ik_solutions[i];
-    trajectory_pt_timed.time_from_start = ros::Duration(running_time);
+    trajectory_pt_timed.time_from_start = rclcpp::Duration(running_time);
```

Comment:
Doesn't really matter since `running_time` is `0.0`, but also here it's unclear what `running_time` really is and if the constructor is called this way intentionally. I'd prefer `rclcpp::Duration::from_seconds(running_time)`.

Reply:
`running_time` is incremented by `display_time`.

Reply:
I've added  `rclcpp::Duration::from_seconds(running_time)`